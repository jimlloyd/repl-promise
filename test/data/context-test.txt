// 'timers' is in context only because it was added when repl was created in unit test.
var deferred = Q.defer();
var timeoutObject = timers.setTimeout(function(){
    deferred.resolve({done: true});
}, 50);

deferred.promise
