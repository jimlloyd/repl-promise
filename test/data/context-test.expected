node > // 'timers' is in context only because it was added when repl was created in unit test.
node > var deferred = Q.defer();
node > var timeoutObject = timers.setTimeout(function(){
...     deferred.resolve({done: true});
... }, 50);
node > deferred.promise
{ done: true }
node > 